staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/dispatcher.go

```diff
@@ -62,30 +62,54 @@ func newValidatingDispatcher(p *Plugin) func(cm *webhookutil.ClientManager) gene
 	}
 }
 
+var _ generic.VersionedAttributeAccessor = &versionedAttributeAccessor{}
```

Comment:
Can this code be moved to the generic webhook package, and shared between mutating & validating?

Reply:
Ah, maybe the caching logic is different enough that it's not worth it... I'll leave it up to you.

Reply:
Yeah the logic is different and the specific implementation of validating/mutating so I think it makes sense with each of them